ethabi/src/lib.rs

```diff
@@ -49,6 +50,7 @@ pub use param::Param;
 pub use log::{Log, RawLog, LogParam, ParseLog, LogFilter};
 pub use event::Event;
 pub use event_param::EventParam;
+pub use tuple_params::TupleParams;
```

Comment:
Let's handle `TupleParam` like `EventParam` — singular. The collection we can trivially express via `Vec<TupleParam>`.